operations/observability/mixins/workspace/rules/satellite/servicemonitor.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
Please refer to https://github.com/gitpod-io/ops/pull/6775/files#r1025318855, I think it makes sense to define this in the ops repo (along with other workspace-cluster servicemonitors), so that we're consistent with other service monitors.